test_runner/regress/test_endpoint_crash.py

```diff
@@ -23,3 +24,16 @@ def test_endpoint_crash(neon_env_builder: NeonEnvBuilder, sql_func: str):
     endpoint.safe_psql("CREATE EXTENSION neon_test_utils;")
     with pytest.raises(Exception, match="This probably means the server terminated abnormally"):
         endpoint.safe_psql(f"SELECT {sql_func}();")
+
+
+@pytest.mark.skipif(not WITH_SANITIZERS, reason="Sanitizers are not enabled, skipping")
```

Comment:
I think we need to run this test always, but it should crash only with sanitisers enabled and pass without them

Reply:
Then maybe it's worth to add this test case (SELECT) to some other test? Creating the neon environment just to perform one simple SELECT seems too expensive to me.

Reply:
> Then maybe it's worth to add this test case (SELECT) to some other test?

In other tests, it'll be surprising to have a totally unrelated `SELECT`.

> Creating the neon environment just to perform one simple SELECT seems too expensive to me.

That's ok to do to test it works.
If you prefer, you can limit the test scope by Postgres version and run only for the default Postgres with `@run_only_on_default_postgres` decorator.

Reply:
Your comment reminded me that we implemented support for sanitizers only for v17, so I added @run_only_on_postgres decorator. As to running the test against a non-sanitized build, I can't imagine a scenario when such a test would fail, but still have done this way.